neofetch

```diff
@@ -93,6 +93,9 @@ get_distro() {
                     "tiny") distro="${distro//version*}" ;;
                 esac
 
+            elif type -p tazpkg >/dev/null; then
+                distro="SliTaz $(< /etc/slitaz-release)"
```

Comment:
The problem is lsb_release showing "Fedora", this is the same issue as MX and AntiX. :/